specs/TransportProtocols.md

```diff
@@ -71,50 +62,17 @@ foo: boz
 
 TBD: Keep Alive - Should it be done at this level?
 
-For example, when sending the following frames (`\n` indicates the actual Line Feed character, not an escape sequence):
-
-* Type=`Text`, "Hello\nWorld"
-* Type=`Binary`, `0x01 0x02`
-* Type=`Close`, `<<no body>>`
-
-The encoding will be as follows
-
-```
-data: T
-data: Hello
-data: World
-
-data: B
-data: AQI=
-
-data: C
-
-[Connection Terminated at this point]
-```
-
-This transport will buffer incomplete frames sent by the server until the full message is available and then send the message in a single frame.
-
 ## Long Polling (Server-to-Client only)
 
 Long Polling is a server-to-client half-transport, so it is always paired with HTTP Post. It requires a connection already be established using the `OPTIONS [endpoint-base]` request.
 
-Long Polling requires that the client poll the server for new messages. Unlike traditional polling, if there are no messages available, the server will simply block the request waiting for messages to be dispatched. At some point, the server, client or an upstream proxy will likely terminate the connection, at which point the client should immediately re-send the request. Long Polling is the only transport that allows a "reconnection" where a new request can be received while the server believes an existing request is in process. This can happen because of a time out. When this happens, the existing request is immediately terminated with status code `204 No Content`. Any messages which have already been written to the existing request will be flushed and considered sent.
+Long Polling requires that the client poll the server for new messages. Unlike traditional polling, if there is no data available, the server will simply wait for messages to be dispatched. At some point, the server, client or an upstream proxy will likely terminate the connection, at which point the client should immediately re-send the request. Long Polling is the only transport that allows a "reconnection" where a new request can be received while the server believes an existing request is in process. This can happen because of a time out. When this happens, the existing request is immediately terminated with status code `204 No Content`. Any messages which have already been written to the existing request will be flushed and considered sent.
 
-Since there is such a long round-trip-time for messages, given that the client must issue a request before the server can transmit a message back, Long Polling responses contain batches of multiple messages. Also, in order to support browsers which do not support XHR2, which provides the ability to read binary data, there are two different modes for the polling transport.
+In order to support browsers which do not support XHR2, which provides the ability to read binary data, there are two different modes for the polling transport.
```

Comment:
Is this still something we care about at this layer? I think the format of the data is negotiated outside this protocol now (JSON/ProtoBuf). The `Content-Type`s would be different, but I think that's actually outside this protocol.